CrowderApp/app/src/main/java/com/example/crowderapp/controllers/ExperimentHandler.java

```diff
@@ -1,33 +1,90 @@
 package com.example.crowderapp.controllers;
 
-import com.example.crowderapp.models.Experiment;
+import android.location.Location;
 
-public class ExperimentHandler {
+import com.example.crowderapp.models.Experiment;
```

Comment:
Is it common practice to put import statements inside the class? Not sure if this will compile...